integration/networktest/traffic/traffic.go

```diff
@@ -0,0 +1,42 @@
+package traffic
```

Comment:
I think this needs a slightly better name and a readme.

Reply:
Added a readme but can't really think of a better name. Some ideas: `load`, `simulation`, `useractivity`, `simtraffic`, `usertraffic`?